.github/workflows/R-CMD-check.yaml

```diff
@@ -20,30 +20,99 @@ jobs:
         config:
           - {os: macos-latest,   r: 'release'}
           - {os: windows-latest, r: 'release'}
-          - {os: ubuntu-latest,   r: 'devel', http-user-agent: 'release'}
+          # - {os: ubuntu-latest,   r: 'devel', http-user-agent: 'release'}
           - {os: ubuntu-latest,   r: 'release'}
-          - {os: ubuntu-latest,   r: 'oldrel-1'}
+          # - {os: ubuntu-latest,   r: 'oldrel-1'}
 
     env:
       GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
       R_KEEP_PKG_SOURCE: yes
+      RENV_PATHS_ROOT: ~/.local/share/renv
 
     steps:
-      - uses: actions/checkout@v3
+      - name: Install rirods
+        uses: actions/checkout@v3
+        with:
+          path: rirods
+
+      - name: Install irods_demo
+        if: startsWith(runner.os, 'Linux')
+        uses: actions/checkout@v3
+        with:
+          repository: irods/irods_demo
+          path: irods_demo
 
-      - uses: r-lib/actions/setup-pandoc@v2
+      - name: Setup iRODS demo
+        if: startsWith(runner.os, 'Linux')
+        run: |
+          cd ./irods_demo
+          git submodule update --init
+          docker-compose up -d
```

Comment:
i think this can just be 

`docker-compose up -d nginx-reverse-proxy`

No need to stand up the other 4-5 containers in the demo.